templates/csi-templates/csi-cephfs-secret.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
```

Comment:
Please add '---' at top f the file

Reply:
@RazTamir the "---" is there in line 1. Have I added it wrongly ? Please let me know if there's a syntactical error here which I have overlooked. :)

Reply:
Oops (: